src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3899,4 +3816,153 @@ bool ViewProviderSketch::isInEditMode() const
 {
     return editCoinManager != nullptr;
 }
+void ViewProviderSketch::generateContextMenu() {
+    int selectedEdges = 0;
+    int selectedLines = 0;
+    int selectedConics = 0;
+    int selectedPoints = 0;
+    int selectedConstraints = 0;
+
+    Gui::MenuItem geom;
```

Comment:
I'm not sure if this name is fitting here, maybe `menu` would be better? `geom` comes from `geometry` an this variable does not represent geometry. I know that it is copied from another place, but it probably would be wise to change it now.

@PaddleStroke you have more experience with codebase - so if this is some convention in the code that I'm simply not aware of please intervene

Reply:
changed

Reply:
The name is not great indeed. I did not paid attention because this was like this before. But yes name 'menu' would be better.